sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Added support for BlobContainerClient.OpenWriteAsync().
 - Fixed bug where BlobSasBuilder.SetPermissions(string rawPermissions) was not properly handling the Permanent Delete ('y') and set Immutability Policy ('i') permissions.
 - Fixed a bug where BlobClient.Upload() and UploadAsync() would result in a NullReferenceException when trying to overwrite a blob with client-side encryption enabled
+- Fixed a bug where BlobBaseClient.DownloadTo() would result in an ObjectDisposedException on .NET Framework in certain network conditions.
```

Comment:
Let's hold off on merging this until the current release is complete, this changlog line will need to go in the next release, the section for which hasn't been added to the changlog yet.